test/tests/functional/basic_single_server_auth_tests.js

```diff
@@ -329,7 +381,7 @@ describe.skip('Basic single server auth tests', function() {
     }
   });
 
-  it('should correctly authenticate server using scram-sha-1 using connect auth then logout', {
+  it.skip('should correctly authenticate server using scram-sha-1 using connect auth then logout', {
```

Comment:
what's going on here? should we perhaps skip it only for <3.7.3?

Reply:
This  test is just broken in general, before I even made changes.